applications/openshift/kubelet/kubelet_configure_tls_cipher_suites_master/rule.yml

```diff
@@ -0,0 +1,70 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+platform: ocp4
+
+title: "Ensure that the Kubelet only makes use of Strong Cryptographic Ciphers"
+
+description: |-
+  Ensure that the Kubelet is configured to only use strong cryptographic ciphers.
+  To set the cipher suites for the kubelet, create new or modify existing
+  <tt>KubeletConfig</tt> object along these lines, one for every
+  <tt>MachineConfigPool</tt>:
+    <pre>
+    apiVersion: machineconfiguration.openshift.io/v1
+    kind: KubeletConfig
+    metadata:
+       name: kubelet-config-$pool
+    spec:
+        machineConfigPoolSelector:
+            matchLabels:
+                pools.operator.machineconfiguration.openshift.io/$pool_name: ""
+        kubeletConfig:
+          tlsCipherSuites:
+          - TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+          - TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
+          - TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+          - TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
+    </pre>
+  In order to configure this rule to check for an alternative cipher, both var_kubelet_tls_cipher_suites_regex
+  and var_kubelet_tls_cipher_suites have to be set
+rationale: |-
+  TLS ciphers have had a number of known vulnerabilities and weaknesses,
+  which can reduce the protection provided by them. By default Kubernetes
+  supports a number of TLS ciphersuites including some that have security
+  concerns, weakening the protection provided.
+
+severity: medium
+
+#identifiers:
+#    cce@ocp4:
```

Comment:
nit: feel free to fix this in a follow up commit but we should have CCE identifiers for all rules